### ISSUE TITLE
Don't re.escape() cmdline path args in tests

### ISSUE THREAD

**Issue by userzimmermann** [coala/coala#5020]
2nd step after https://github.com/coala/coala/issues/4890 for getting rid of inconveniently used `re.escape()`s

Since https://github.com/coala/coala/pull/4883 coala cmdline args can have single-backslash path separators on Windows, which were formerly doubled in the tests by using `re.escape` to avoid their removal by coala's settings handling (always interpreting backslashes as escape characters)

Occurencies of those obsolete `re.escape()`s can be found via https://github.com/coala/coala/pull/5019#discussion_r159142870 https://github.com/coala/coala/pull/5019#discussion_r159142883 https://github.com/coala/coala/pull/5019#discussion_r159142889

Maybe there are even more... :)

cc @Makman2

**Comment by userzimmermann** [comment]
Blocked by https://github.com/coala/coala/issues/5022

**Comment by userzimmermann** [comment]
@ishanSrt Closing this in favor of #5022 and related PR https://github.com/coala/coala/pull/5023 which implicitly integrates all changes for solving this
